integrationtests/src/test/java/io/kroxylicious/proxy/multitenant/MultiTenantIT.java

```diff
@@ -414,27 +406,27 @@ private static void runConsumerInOrderToCreateGroup(KroxyliciousTester tester, S
         }
     }
 
-    private static void verifyConsumerGroupsWithList(Admin admin, Set<String> expectedGroupIds) throws Exception {
+    private void verifyConsumerGroupsWithList(Admin admin, Set<String> expectedGroupIds) throws Exception {
         var groups = admin.listConsumerGroups().all().get().stream().map(ConsumerGroupListing::groupId).toList();
         assertThat(groups).containsExactlyInAnyOrderElementsOf(expectedGroupIds);
     }
 
-    private static void verifyTransactionsWithList(Admin admin, Set<String> expectedTransactionalIds) throws Exception {
+    private void verifyTransactionsWithList(Admin admin, Set<String> expectedTransactionalIds) throws Exception {
         var transactionalIds = admin.listTransactions().all().get().stream().map(TransactionListing::transactionalId).toList();
         assertThat(transactionalIds).containsExactlyInAnyOrderElementsOf(expectedTransactionalIds);
     }
 
-    private static void verifyTenant(Admin admin, String... expectedTopics) throws Exception {
+    private void verifyTenant(Admin admin, String... expectedTopics) throws Exception {
         var listTopicsResult = admin.listTopics();
 
         var topicListMap = listTopicsResult.namesToListings().get();
-        assertEquals(expectedTopics.length, topicListMap.size());
+        assertThat(expectedTopics.length).isEqualTo(topicListMap.size());
         Arrays.stream(expectedTopics)
                 .forEach(expectedTopic -> assertThat(topicListMap).hasEntrySatisfying(expectedTopic, allOf(matches(TopicListing::name, expectedTopic))));
 
         var describeTopicsResult = admin.describeTopics(TopicNameCollection.ofTopicNames(Arrays.stream(expectedTopics).toList()));
         var topicDescribeMap = describeTopicsResult.allTopicNames().get();
-        assertEquals(expectedTopics.length, topicDescribeMap.size());
+        assertThat(expectedTopics.length).isEqualTo(topicDescribeMap.size());
```

Comment:
```suggestion
        assertThat(expectedTopics).hasSize(topicListMap.size());
```
or

```suggestion
        assertThat(expectedTopics).hasSameSizeAs(topicListMap);
```

Reply:
As mentioned above, `expectedTopics` is a vararg and therefore an array with no size property, so this doesn't work.